package.json

```diff
@@ -46,9 +46,9 @@
   "devDependencies": {
```

Comment:
I pinned some versions because of bugs/incompatibilities. Please see description for a more detailed explanation